io.github.dagargo.Elektroid.yaml

```diff
@@ -0,0 +1,42 @@
+id: io.github.dagargo.Elektroid
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: elektroid.sh
+finish-args:
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=all"
+rename-desktop-file: elektroid.desktop
+rename-appdata-file: elektroid.appdata.xml
+rename-icon: elektroid
+copy-icon: true
+separate-locales: false
+modules:
+  - name: libzip
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://libzip.org/download/libzip-1.9.2.tar.gz
+        sha256: fd6a7f745de3d69cf5603edc9cb33d2890f0198e415255d0987a0cf10d824c6f
+    # Disable installation of libzip tools in /app/bin:
+    config-opts: ['-DBUILD_TOOLS=OFF']
```

Comment:
you probably need also a cleanup rule for the development files like includes and pkg-config.

Reply:
this comment wasn't addressed.

Reply:
I intentionally leave this for later because I didn't understand what you meant and I was expecting to face the issue on the way. But nothing has arised and everything looks good to me.

Could you elaborate on the need for the cleanup or point me in the right direction?

BTW, thanks for taking the time to review all these things.

Reply:
You should add something along `cleanup: ["/include"]` to remove unnecessary files. You can look up examples using GitHub search scoped to the Flathub org.

Reply:
`/share/man`, `/lib/cmake`, `/lib/pkgconfig` too, but given there's not much to save here, feel free to address this post-merge.